docs/reference/setup/install/getting-started-tls-docker.asciidoc

```diff
@@ -0,0 +1,171 @@
+[[getting-started-tls-docker]]
+=== Getting started with TLS and Elasticsearch
+
+Starting with version 6.0.0, x-pack licensed for security (gold, platinum or enterprise subscriptions) https://www.elastic.co/guide/en/elasticsearch/reference/6.0/breaking-6.0.0-xes.html[requires SSL/TLS] encryption for the transport networking layer.
```

Comment:
Is "X-Pack Security" cleaner than "X-Pack licenced for Security"?

Reply:
Addressed in d258de65303b51ed3e19539521fc8a69bff6badf